internal/pkg/levee/levee.go

```diff
@@ -75,11 +72,10 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func reportSourcesReachingSink(pass *analysis.Pass, sources []*source.Source, instr ssa.Instruction, propagations map[ssa.Node]propagation.Propagation) {
-	for _, s := range sources {
-		prop := propagations[s.Node]
+func reportSourcesReachingSink(pass *analysis.Pass, instr ssa.Instruction, propagations map[*source.Source]propagation.Propagation) {
```

Comment:
should this be reportSourcesReachingASink?

Reply:
I personnally think `ReachingSink` is fine, to me having the `A` makes this name slightly less readable.

Reply:
Looking at the signature more closely prompted me to make the following changes:
* `instr` --> `sink`
* Swapped the last two arguments so that the order of arguments mirrors the order in the name (sources before sink)